### DISCUSSION TITLE
How to display not linkable sids in console

### DISCUSSION THREAD

**Discussion by juallom** [twilio-labs/paste#657] [Q&A] [answered]
Hi team QQ

I've seen throughout the console that when a sid is not an anchor it is displayed in a different font-family.
I have followed this pattern in the new sip-trunking call instance page using PASTE:

```
<Text as="span" fontFamily="fontFamilyCode">
   {callSid}
</Text>
```
Is this correct or should I do it differently?

![Screen Shot 2020-08-13 at 6 10 58 PM](https://user-images.githubusercontent.com/7203420/90160019-7cae1000-dd91-11ea-8406-9312426f335b.png)

**Comment by corinne-nagel** [comment, marked as answer]
Hi Jaun! That looks good to us 👍
